api/client/service.proto

```diff
@@ -32,6 +32,9 @@ message ExecutionRequest {
 message ExecutionResponse {
   Output output = 1;
   google.rpc.Status error_detail = 7;
+  // Unique id for lookup purposes. This will be taken from CommandLineOptions if set,
+  // if it is not set there it will be auto-generated.
```

Comment:
Shall we say that it is an opaque identifier whose format can change at any time? I.e. users shouldn't depend on the format in any way.

Just to give ourselves some flexibility if we need it.

Reply:
Done: 8f8a5da